lib/ae_mdw/names.ex

```diff
@@ -72,6 +72,22 @@ defmodule AeMdw.Names do
   @min_bin Util.min_bin()
   @max_bin Util.max_256bit_bin()
 
+  @spec count_names(state(), query()) :: {:ok, non_neg_integer()} | {:error, reason()}
+  def count_names(state, query) do
+    query
+    |> Util.convert_params(&convert_param/1)
+    |> case do
+      {:ok, %{owned_by: owned_by}} ->
+        {:ok, count_active_names(state, owned_by)}
+
+      {:ok, %{}} ->
+        {:ok, count_active_names(state, nil)}
+
+      {:error, _} ->
```

Comment:
`{:error, _reason}`. Also, can't we just return this same error and use a with expresion?

Reply:
No, because it raises a warning that a `with` with one clause and an else, should be a `case`